test/cloudtest/pkg/tests/all_cluster_fail_test.go

```diff
@@ -135,14 +138,16 @@ func TestClusterInstancesOnFailShellRunner(t *testing.T) {
 
 	report, err := commands.PerformTesting(testConfig, &testValidationFactory{}, &commands.Arguments{})
 	g.Expect(err.Error()).To(Equal("there is failed tests 1"))
+	foundFailTest := false
 
 	for _, t := range report.Suites[0].TestCases {
 		if t.Name == "_fail" {
 			g.Expect(t.Failure).NotTo(Equal(BeNil()))
 			g.Expect(strings.Contains(t.Failure.Contents, ">>>Running on fail script<<<")).To(Equal(true))
-
+			foundFailTest = true
```

Comment:
Is it necessary here? I assume at least cycle can be finished here

Reply:
Yes, it is necessary. It excludes the case of the renaming of the failing test.

Reply:
So could you fail test at the moment when you found renamed test